File: WindowsFormsApp1/Models/SuppliersContactTitle.cs
using System;
using System.Collections.Generic;

namespace WindowsFormsApp1.Models
{
    public partial class SuppliersContactTitle
    {
        public SuppliersContactTitle()
        {
            Suppliers = new HashSet<Suppliers>();
        }

        public int SuppliersTitleId { get; set; }
        public string Title { get; set; }

        public virtual ICollection<Suppliers> Suppliers { get; set; }
    }
}

File: Converted/Models/SuppliersContactTitle.vb
Imports System
Imports System.Collections.Generic

Namespace WindowsFormsApp1.Models
	Partial Public Class SuppliersContactTitle
		Public Sub New()
			Suppliers = New HashSet(Of Suppliers)()
		End Sub

		Public Property SuppliersTitleId() As Integer
		Public Property Title() As String

		Public Overridable Property Suppliers() As ICollection(Of Suppliers)
	End Class
End Namespace
